### PR TITLE
scroll and resize are part of CSSOM-VIEW

### LINKED ISSUE #147: event 'scroll' type is no longer part of UIEvents
https://w3c.github.io/uievents/#event-types-list doesn't have scroll anymore

### PR DESCRIPTION
**Pull request by plehegar**
Fixes #147

### CODE REVIEW

File: single-page.bs [hunk 1]
```diff
@@ -376,10 +376,19 @@ urlPrefix: https://www.w3.org/TR/CSP3/#; type: dfn; spec: CSP
...
 <!-- ********************************** CSSOM-VIEW ********************************************* -->
 
-urlPrefix: http://www.w3.org/TR/cssom-view/#; type: interface; spec: CSSOM-VIEW;
-    text: screen;
+urlPrefix: http://www.w3.org/TR/cssom-view/#; spec: CSSOM-VIEW;
+    text: screen; type: interface
+    urlPrefix: eventdef-document-; type: event;
+        text: scroll
+        text: resize
+    text: run the resize steps; type: dfn
```
Comment (arronei) [inline single-page.bs:384]: You might want to pull up the type: dfn a level and nest all these text: items under that dfn section

```
type: dfn;
    text: run the resize steps
    text: run the scroll steps
```

etc...

Comment (travisleithead) [inline single-page.bs:384]: I'm not sure if Arron's suggestion will work. I'm fine with the original change it as is.

**Comment by travisleithead** [comment]
@plehegar thoughts? If you're fine with the text then I can merge it in.

**Comment by plehegar** [comment]
we can merge this one and I'll see to improve things

**Comment by travisleithead** [comment]
Sounds good. Thanks!

**Pull request merged**